drivers/rn2xx3/rn2xx3_getset.c

```diff
@@ -0,0 +1,363 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_rn2xx3
+ * @{
+ * @file
+ * @brief       Implementation of get and set functions for RN2XX3
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @}
+ */
+
+#include <string.h>
+#include <errno.h>
+
+#include "assert.h"
+#include "rn2xx3.h"
+#include "rn2xx3_internal.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+char cmd[32];
+static const char * mac = "mac";
+static const char * get = "get";
+static const char * set = "set";
+
+/* internal helpers */
+static uint8_t _get_uint8_value(rn2xx3_t *dev, const char* command)
+{
+    size_t p = snprintf(dev->cmd_buf, sizeof(dev->cmd_buf) - 1,
+                        "%s %s %s", mac, get, command);
+    dev->cmd_buf[p] = 0;
+
+    DEBUG("CMD: %s\n", dev->cmd_buf);
```

Comment:
There is already DEBUG in rn2xx3_write_cmd() to show the CMD. The ones here in rn2xx3_getset.c can be removed.

May I suggest to do the RESP debug in rn2xx3_write_cmd() as well? Then we can make this module (rn2xx3_getset.c) nice and clean.